.gitignore

```diff
@@ -1,6 +1,7 @@
 venv/
 build/
 *.charm
+*.crt
```

Comment:
This is bad workflow to ignore the files created in the test.

It would be way better to put them in a [TemporaryDirectory](https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory) that will be fully when going out of the context manager.
Or event directly a `TemporaryFile`.
I can help with that if you want :)

Reply:
that would be a huge help, yes please 🙇‍♀️

Reply:
Ok, do you want me to commit on your branch or should we talk about it during our 1-1?

Reply:
please commit directly! :)

Reply:
Ok :)
As I thought, with pytest it's even easier since it provides a fixture just for that.

Reply:
Done